Ginger/GingerCore/Drivers/Appium/SeleniumAppiumDriver.cs

```diff
@@ -443,21 +490,28 @@ private DriverOptions GetCapabilities()
 
             //see http://appium.io/slate/en/master/?csharp#appium-server-capabilities for full list of capabilities values
 
+            //Device capabilities
+            if (string.IsNullOrEmpty(DeviceID) == false)
```

Comment:
Codacy found an issue: [Remove the unnecessary Boolean literal(s).](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6113827)